### ISSUE TITLE
Herbeden node

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10711]
**Preferred term label:**
Herbeden node

**Synonyms**

**Definition (free text, please give PubMed ID)**
Periarticular osteophyte formation around the DIP joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal interphalangeal (DIP) joint arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
[Heberden's node HP:0012313](https://hpo.jax.org/browse/term/HP:0012313)

I will change the primary label to Heberden node

**Comment by pnrobinson** [comment]
@MickeySegal 
Note that your NTR included a type - you may want to check that Heberden (not Herbeden) is spelled correctly in other resources.
